test/fixtures/user-integration-app/server/server.js

```diff
@@ -0,0 +1,10 @@
+var loopback = require('../../../../index');
+var boot = require('loopback-boot');
+var app = module.exports = loopback();
+app.enableAuth();
+boot(app, __dirname);
+app.use(loopback.token({model: app.models.AccessToken}));
+var apiPath = '/api';
+app.use(apiPath, loopback.rest());
+app.use(loopback.urlNotFound());
+app.use(loopback.errorHandler());
```

Comment:
> The purpose of the user.integration.js is to test a LB app's built-in user model.

As you wrote yourself, there is nothing related to loopback-boot in the scenario you are testing.

It is a good practice to write the test code in such way that it's easy to quickly understand what is the scenario being tested. The test scenario should be also as minimal as possible and include only things that are absolutely necessary for the test.

By using a fixture defined in several files living outside of the test file, you are making it difficult to understand what is exactly being tested. By including lots of unrelated stuff in your fixture, you make it difficult to see what is the important (test-specific) part and what is just boilerplate. Last but not least, all these extra unneeded bits are making the maintenance more expensive.

This is already visible in this pull request, as the `server/server.js` file is not following the project structure as scaffolded by recent versions of loopback-workspace.

To test the built-in user model, you need just few lines of code:

```js
var app = loopback();
app.dataSource('db', { connector: 'memory' });
app.model(loopback.User, { dataSource: 'db' });
app.model(loopback.AccessToken, { dataSource: 'db' });
app.model(loopback.ACL, { dataSource: 'db' });
app.model(loopback.Role, { dataSource: 'db' });
app.model(loopback.RoleMapping, { dataSource: 'db' }); // this may be not necessary

app.enableAuth();

app.use(loopback.rest());
request = supertest(app);
```

I admit this setup is not as short as it should be, let's discuss the best API in https://github.com/strongloop/loopback/pull/1323 and keep the code above for the purposes of this PR.